tests/find_toolbar/test_find_in_pdf.py

```diff
@@ -0,0 +1,41 @@
+import logging
+from time import sleep
+
+from selenium.webdriver import Firefox
+from selenium.webdriver.common.keys import Keys
+
+from modules.browser_object import FindToolbar
+from modules.page_base import BasePage
+from modules.page_object import GenericPdf
+
+WORD_SEGMENT = "authori"
+MATCH_TWO_COTEXT = "representative must complete"
```

Comment:
Nb: I know this looks like "context" but I did actually mean "cotext": "the words in the environment immediately surrounding a word or phrase"